app/scripts/dateformat.js

```diff
@@ -0,0 +1,226 @@
+/*
```

Comment:
As we discussed during the meeting today, make this library a bower dependency and remove it from the repo.